test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -73,7 +73,7 @@ static const std::string filter_config_name = "scooby.dooby.doo";
 
 class HttpFilterTest : public testing::Test {
 protected:
-  void initialize(std::string&& yaml) {
+  void initialize(std::string&& yaml, bool will_call_upstream_host = true) {
```

Comment:
Could you explain why it is not called? Do you still need this if you move the code to closeStream?

Reply:
It was added by mistake. Removed.